Sources/ComposableCoreLocation/Interface.swift

```diff
@@ -282,142 +275,79 @@ public struct LocationManager {
 
-  public var requestLocation: (AnyHashable) -> Effect<Never, Never>
+  public var requestLocation: () -> Effect<Never, Never>
 
   @available(macOS, unavailable)
-  public var requestWhenInUseAuthorization: (AnyHashable) -> Effect<Never, Never>
+  public var requestWhenInUseAuthorization: () -> Effect<Never, Never>
 
-  public var requestTemporaryFullAccuracyAuthorization: (AnyHashable, String)
+  public var requestTemporaryFullAccuracyAuthorization: (String)
   -> Effect<Never, Never>
 
-  public var set: (AnyHashable, Properties) -> Effect<Never, Never>
+  public var set: (Properties) -> Effect<Never, Never>
 
   @available(tvOS, unavailable)
   @available(watchOS, unavailable)
   public var significantLocationChangeMonitoringAvailable: () -> Bool
 
   @available(tvOS, unavailable)
   @available(watchOS, unavailable)
-  public var startMonitoringForRegion: (AnyHashable, Region) -> Effect<Never, Never>
+  public var startMonitoringForRegion: (Region) -> Effect<Never, Never>
 
   @available(tvOS, unavailable)
   @available(watchOS, unavailable)
-  public var startMonitoringSignificantLocationChanges: (AnyHashable) -> Effect<Never, Never>
+  public var startMonitoringSignificantLocationChanges: () -> Effect<Never, Never>
 
   @available(macOS, unavailable)
   @available(tvOS, unavailable)
   @available(watchOS, unavailable)
-  public var startMonitoringVisits: (AnyHashable) -> Effect<Never, Never>
+  public var startMonitoringVisits: () -> Effect<Never, Never>
 
   @available(macOS, unavailable)
   @available(tvOS, unavailable)
-  public var startUpdatingHeading: (AnyHashable) -> Effect<Never, Never>
+  public var startUpdatingHeading: () -> Effect<Never, Never>
 
   @available(tvOS, unavailable)
-  public var startUpdatingLocation: (AnyHashable) -> Effect<Never, Never>
+  public var startUpdatingLocation: () -> Effect<Never, Never>
 
   @available(tvOS, unavailable)
   @available(watchOS, unavailable)
-  public var stopMonitoringForRegion: (AnyHashable, Region) -> Effect<Never, Never>
+  public var stopMonitoringForRegion: (Region) -> Effect<Never, Never>
 
   @available(tvOS, unavailable)
   @available(watchOS, unavailable)
-  public var stopMonitoringSignificantLocationChanges: (AnyHashable) -> Effect<Never, Never>
+  public var stopMonitoringSignificantLocationChanges: () -> Effect<Never, Never>
 
   @available(macOS, unavailable)
   @available(tvOS, unavailable)
   @available(watchOS, unavailable)
-  public var stopMonitoringVisits: (AnyHashable) -> Effect<Never, Never>
+  public var stopMonitoringVisits: () -> Effect<Never, Never>
 
   @available(macOS, unavailable)
   @available(tvOS, unavailable)
-  public var stopUpdatingHeading: (AnyHashable) -> Effect<Never, Never>
-
-  public var stopUpdatingLocation: (AnyHashable) -> Effect<Never, Never>
-
-  @available(iOS 14.0, tvOS 14.0, watchOS 7.0, macOS 11.0, macCatalyst 14.0, *)
-  public func accuracyAuthorization(id: AnyHashable) -> AccuracyAuthorization? {
```

Comment:
I just got rid of these overloads...there's no way to really deprecate things. The compiler will just let them know they are passing an argument to a function that takes no arguments.